### ISSUE TITLE
project update

### ISSUE THREAD

**Issue by npomfret** [elliottsj/react-native-healthkit#3]
Hi - thanks for putting this project together.  If it's working could you put a basic readme on here to briefly describe how t use it?

**Comment by elliottsj** [comment]
Hey @npomfret. I'm no longer using this project (and it's obviously missing most of the HealthKit APIs), but I'm open to transferring this project over to someone who can maintain it. Added a README: https://github.com/elliottsj/react-native-healthkit/blob/master/README.md

**Comment by grabbou** [comment]
Hey @elliottsj, we are looking into using this internally with @alimek - if that's still looking for a maintainer, happy to give it a go!

**Comment by elliottsj** [comment]
Hey @grabbou, I'm happy to transfer this over to you! Would you like me to transfer it to your own GitHub account? I've now added https://www.npmjs.com/~grabbou to the owners on the npm package.

**Comment by grabbou** [comment]
Yeah, I think that works :) thanks for the awesome work so far!

**Comment by elliottsj** [comment]
Alright, transfer requested!

**Comment by mintuz** [comment]
Hello any update on what is happening with the project. Looking for HealthKit integration for my React Native app but not looking to depend on a module if it's not being maintained.

**Comment by jpmonette** [comment]
@grabbou Still working on this? I'd be happy to take maintain it going forward.

**Comment by grabbou** [comment]
Yeah, to be honest, we no longer work with the client who was willing to sponsor further development of this library.

That said, I am happy to transfer you the ownership if you are interested in moving this library forward. Please let me know and I'll do it right away.
